plugins/explain.py

```diff
@@ -83,6 +83,16 @@ class Explain(BotPlugin):
                      'newcomer Pull Request\n- merge a difficulty/low Pull '
                      'Request\n- review at least a difficulty/low or higher '
                      'Pull Request'
+        'do not ask to ask': 'This is a lazy way of asking your question '
+                             'because  you\'re asking for more than what '
+                             'you think you\'re asking. The solution is not '
+                             'to ask to ask, but just to ask by actaully '
+                             'formalizing it in words. Someone who is idling '
+                             'on the channel and only every now and then '
```

Comment:
"and only **occasionally** glances [...]" sounds a bit more natural here. Otherwise the "and"s sound a little weird.

Reply:
Okay. I shall do this.